services/core-api/app/api/projects/major_mine_application/resources/major_mine_application_list.py

```diff
@@ -36,7 +36,6 @@ def post(self, project_guid):
 
         try:
             major_mine_application.save()
-            major_mine_application.send_mma_submit_email()
```

Comment:
Is there never a reason that the post would need to send a submission email and by putting the email in the put is there the possibility the email is sent more than once?

Reply:
The requirements are that it should be sent when a MMA is submitted(not save draft)(PUT only, can't be reached with a POST). It can be sent more than once if an application gets set to "Changes Requested"(adding in another ticket) which restarts the flow all over again in terms of (re)submission. UX didn't seem concerned about it at the time.

Reply:
K, and there is no way that the POST would have status REC and only draft? is the draft a required stage?

Reply:
Draft is required because you cannot advance to Step 3 if the application has not been saved as a draft. In terms of the POST, the status code is hardcoded in the payload so there is no way it could be "REC".